development/index.md

```diff
@@ -38,40 +52,126 @@ There are three types of issue used for planning work:
  - **Task**: a piece of work that isn't necessarily tied to a specific Epic or Story.
    For example, items related to technical debt or house-keeping chores.
 
-A planning meeting is held at the start of the release. In this meeting the team
-discuss the current backlog of stories, tasks and bugs to propose items to be added
-to the release. All items assigned to a milestone should have an assigned owner.
 
-### Retrospective
+Whenever an issue is raised, it will be reviewed by the Product Owner/CTO and added
+to the Product Board for prioritisation and planning. The exception to this are
+tasks/bugs related to work already in progress and that need to be addressed in
+the current milestone. They should be added to the Development Board and current
+milestone directly.
 
-The day after each FlowForge release, always a Friday, a meeting is scheduled
-by the CTO. This meeting includes 3 parts: a retrospective on the last release,
-complete scheduling for the next release, and kick off development for the next
-release.
+### Project Boards
```

Comment:
These boards do not work when working on Node-RED mostly, or if there's dependencies between work, it makes it hard to trace on project boards. Is there a solution for this?

Reply:
Where we have an upstream dependency we can raise an issue to track it (labelled 'upstream'). If it is an upstream dependency we can work on directly (eg Node-RED), that lets us track progress in our own board. If it is an upstream dependency we rely on the upstream project delivering themselves, it gives us our own artifact to make as a blocker to our items.